yarn.lock

```diff
@@ -896,6 +896,11 @@
     "@ethersproject/properties" "^5.7.0"
     "@ethersproject/strings" "^5.7.0"
 
+"@fastify/busboy@^2.0.0":
```

Comment:
shall we completely switch over to undici and remove axios, or move axios to `devDependencies`?

Reply:
I've used undici only on the area that were failing with axios (due to the node 20 keepAlive issue)
We could use it more wisely, but it's not very convenient since its API is not as simple as axios, and we'll still need axios on web3 function examples